homeassistant/components/dialogflow/__init__.py

```diff
@@ -84,23 +87,45 @@ class DialogFlowError(HomeAssistantError):
 
 def dialogflow_error_response(message, error):
     """Return a response saying the error message."""
-    dialogflow_response = DialogflowResponse(message["result"]["parameters"])
+    _api_version = get_api_version(message)
+    if _api_version is V1:
```

Comment:
Use `==` for comparison, not `is`. `is` is for pointer equivalency.

Reply:
I intended pointer equivalency here since it's comparing integers, but I'm happy to change it to ==.